README.md

```diff
@@ -1,6 +1,6 @@
 # electronic-prescription-service-api
 
-![Build](https://github.com/NHSDigital/electronic-prescription-service-api/workflows/Build/badge.svg?branch=master)
+[![Build Status](https://dev.azure.com/NHSD-APIM/API%20Platform/_apis/build/status/NHSDigital.electronic-prescription-service-api?branchName=master)](https://dev.azure.com/NHSD-APIM/API%20Platform/_build/latest?definitionId=7&branchName=master) 
```

Comment:
What does this change do?

Reply:
Currently this badge does not exist so does not render, I've updated it to point to the Azure Pipelines build status badge so that we'll get the build status from master present on the README